ui-modules/blueprint-composer/app/components/providers/blueprint-service.provider.js

```diff
@@ -603,16 +616,24 @@ function BlueprintService($log, $q, $sce, paletteApi, iconGenerator, dslService)
 
     function populateLocationFromApiSuccess(entity, data) {
         entity.clearIssues({group: 'location'});
-        entity.location = data.symbolicName;
-        entity.miscData.set('locationName', data.name);
-        entity.miscData.set('locationIcon', data.iconUrl || iconGenerator(data.symbolicName));
+        entity.location = data.yamlHere || data.symbolicName;
```

Comment:
`yamlHere` property feels odd. I would go for just `yaml` or `spec`

Reply:
i wanted something that obviously wouldn't conflict.  it could be `yamlOverride` but to me, `Here` conveys that it is _not_ part of any external spec.